mlproc.py

```diff
@@ -11,56 +11,13 @@
 
 """
 
-import time
+import time, os, signal, sys
```

Comment:
It seems os, signal, sys modules are not used in the code

Reply:
You are right! Sorry! They have been left there from my testing version.